### PR TITLE
Fixed a bug that could cause regexp to issue an error

### PR DESCRIPTION
**Pull request by Ulyanov-programmer**
I changed the regular expression so that it ignores those places where there are inconvenient characters.

### CODE REVIEW

File: index.test.js [hunk 1]
```diff
@@ -608,12 +608,39 @@ test("Save the parent's comment", () => {
...
+  run(
+    `
+div {
+  /* Comment with ^ $ . | ? * + () */
+  &[data-roots-all^=1] * #id .class {}
+}`,
+    '/* Comment with ^ $ . | ? * + () */ div[data-roots-all^=1] * #id .class {}')
+})
+
+// !
```
Comment (Ulyanov-programmer) [inline index.test.js:621]: Problematic added test

File: index.test.js [hunk 2]
```diff
@@ -608,12 +608,39 @@ test("Save the parent's comment", () => {
...
+// test("Save the parent's comment with newline", () => {
```
Comment (Ulyanov-programmer) [inline index.test.js:622]: I added this test to see what happens if there are several rules with comments in the description.

The test does not work - it does not connect selectors for the second nested one. 
I tried to fix it, but in your code, the devil will break my leg. 

Does it make sense to fix it?

File: index.test.js [hunk 3]
```diff
@@ -608,12 +608,39 @@ test("Save the parent's comment", () => {
...
+
+//   /*i2*/
+//   b {} 
+//   /*i3*/
+//   s {} 
+// }`,
+//     `a { /*i*/ } /*i2*/ a b {} /*i3*/ a s {}`
+//   )
+// })
+
```
Comment (Ulyanov-programmer) [inline index.test.js:636]: I added this test to see what happens if there are several rules with comments in the description.

The test does not work - it does not connect selectors for the second nested one. 
I tried to fix it, but in your code, the devil will break my leg. 

Does it make sense to fix it?

**Comment by ai** [comment]
I will release it a few hours later

**Comment by ai** [comment]
Thanks!

**Comment by ai** [comment]
The fix was released in 7.0.1.

Thanks to @Ulyanov-programmer

**Pull request merged**